code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/common/AEPPresentable.kt

```diff
@@ -59,6 +59,14 @@ internal abstract class AEPPresentable<T : Presentation<T>> :
 
     @VisibleForTesting internal val contentIdentifier: Int = Random().nextInt()
 
+    /**
```

Comment:
For some reason Circle CI is not picking up updated coverage  https://app.codecov.io/gh/adobe/aepsdk-core-android/pull/681